### PR TITLE
ephys Parameter now inherits from main Parameter

### PR DESCRIPTION
**Pull request by wvangeit**

### CODE REVIEW

File: bluepyopt/ephys/parameters.py [hunk 1]
```diff
@@ -19,95 +19,47 @@
...
+            value=value,
+            frozen=frozen,
+            bounds=bounds)
+
+    @abstractmethod
+    def instantiate(self, sim=None, icell=None):
+        """Instantiate the parameter in the simulator"""
         pass
 
-    def raise_if_no_value(self):
```
Comment (trialsbiker) [inline bluepyopt/ephys/parameters.py]: why not keep this so the code doesn't have to be duplicated in each function?

Comment (mgeplf) [inline bluepyopt/ephys/parameters.py]: why not keep this so the code doesn't have to be duplicated each time?  If you want the class name, just grab self.__class__.__name__

Comment (wvangeit) [inline bluepyopt/ephys/parameters.py]: It's only two lines. Find it much more readable to explicitly code it than adding another function call. Other parameter types could also have different requirements.

Comment (mgeplf) [inline bluepyopt/ephys/parameters.py]: But it also means one has to write more tests across this error path, and when there's an update, it has to be applied in multiple places.

**Pull request merged**